hack/filter-crd/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+var removeKeys = []string{}
+
+func main() {
+	loadVariant()
+
+	if len(flag.Args()) < 1 {
+		fmt.Println("Usage: filter-crd <CRD YAML file>")
+		return
```

Comment:
nit: log.Fatal here? We should exit non-zero if the args are incorrect I think

Reply:
yeah... good idea